src/adapters/bidderFactory.js

```diff
@@ -233,10 +234,10 @@ export function newBidder(spec) {
                 error: onFailure
               },
               undefined,
-              {
+              Object.assign({}, {
```

Comment:
Tiny optimization: you don't need to `Object.assign({}, defaults, overrides)` here. Since the defaults are defined right here, `Object.assign(defaults, overrides)` should work just fine, and allocate one fewer object.

Definitely shouldn't hold up the merge though.